src/app/hooks/useTransactions.ts

```diff
@@ -16,13 +16,15 @@ export const useTransactions = () => {
         const getTransactionsResponse = await api.getTransactions({
           limit,
         });
-
         const transactions = getTransactionsResponse.transactions.map(
           (transaction) => ({
             ...transaction,
             title: transaction.memo,
-            date: dayjs(transaction.settleDate).fromNow(),
-            timestamp: transaction.settleDate,
+            date: dayjs(
+              transaction.settleDate || transaction.creationDate
+            ).fromNow(),
+            timestamp:
```

Comment:
why is this needed?

Reply:
for pending transaction we don't have settleDate. so we show creationDate there. same is done in go app i think

Reply:
to me the wording is strange, especially the "date" which is not a date